geoportal/geoportailv3_geoportal/static-ngeo/js/olcs/toggle3d.js

```diff
@@ -16,12 +16,51 @@ class Controller {
   /**
    * @ngInject
    * @param {ngeo.olcs.Service} ngeoOlcsService The service.
+   * @param {ngeo.map.BackgroundLayerMgr} ngeoBackgroundLayerMgr Background layer manager.
+   * @param {app.MvtStylingService} appMvtStylingService Mvt styling service.
    */
-  constructor(ngeoOlcsService) {
+  constructor(ngeoOlcsService, ngeoBackgroundLayerMgr, appMvtStylingService) {
     /**
      * @export
      */
     this.manager = ngeoOlcsService.getManager();
+
+    /**
+     * @type{app.Mvtstyling}
+     * @private
+     */
+    this.appMvtStylingService_ = appMvtStylingService;
+
+    /**
+     * @type {ngeo.map.BackgroundLayerMgr}
+     * @private
+     */
+    this.backgroundLayerMgr_ = ngeoBackgroundLayerMgr;
+  }
+
+  luxToggle3d() {
+      const serial = new URLSearchParams(window.location.search).get('serial');
```

Comment:
What happens when the website is directly opened in 3D? I think it is missing a publish.